android/src/play/java/com/dooboolab/RNIap/RNIapModule.kt

```diff
@@ -40,15 +40,15 @@ class RNIapModule(
     ReactContextBaseJavaModule(reactContext),
     PurchasesUpdatedListener {
 
-    private val billingClient: BillingClient = builder.setListener(this)
-        .build()
+    private var billingClient: BillingClient? = null
```

Comment:
Why do we need to set this to null? that's not what the samples in the billingCllient lib suggest